procedures/release-duty/merge-duty/merge_duty.rst

```diff
@@ -376,31 +376,30 @@ Update the releng changelog
 Bump Nightly version and release dates in ShipIt
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-ShipIt currently hard-codes the version of Nightly that's being released, as
-well as the release dates.
+In ShipIt, the Firefox nightly version and the release dates are hard-coded, and a human must update them when a new nightly builds becomes available.
+The nightly builds trigger on a schedule at 10:00 and 22:00 hours UTC and take a few hours to complete.
```

Comment:
We may want to link to .cron.yml, since we occasionally adjust these times and may not update the documentation to match.